src/components/Settings.jsx

```diff
@@ -5,7 +5,7 @@ import { useI18n } from 'cozy-ui/transpiled/react/I18n'
 import { useQuery } from 'cozy-client'
 
 import { AccountContext } from 'src/components/AccountProvider'
-import { buildAccountQuery } from 'src/queries/queries'
+import { buildAccountQuery } from 'src/utils/queries'
```

Comment:
I don't really care where are located the files, but may I ask why this change? Intuitively I do not really expect to find queries defined in an utils folder.

Reply:
Pour regrouper tout ce qui n'est pas composant ou lib dans un même sous-dossier, plutôt que de créer des dossiers à chaque fois. Initialement (via create-cozy-app) les queries sont là-dedans.

Reply:
Pour moi les queries sont totalement légitimes à avoir leur propre répertoire plutôt que d'être dans un dossier "utils" fourre-tout :)

Reply:
Sur ce genre de sujet (doctypes et queries) à mon sens il y a plusieurs approches :
- la subjectivité (chacun peut avoir son avis sur comment on gère l'arborescence des fichiers, ça peut donc être long)
- la cohérence avec l'existant (faire comme sur les autres apps et notamment create-cozy-app)
- les guidelignes (sauf erreur de ma part il n'y a pas ce niveau de détail dans https://github.com/cozy/cozy-guidelines)
- l'ultime bafouille de @Crash-- 

L'idée c'est pas non plus d'y passer des heures... et je pense qu'on est plutôt dans l'ordre du nit là non ?

Bref je vais modifier la PR pour ne laisser que la suppression des trucs inutiles, histoire d'avancer... :unamused:

Reply:
Yes, on est sur du nit :)
Et je pense qu'il y a effectivement une dose non négligeable de subjectivité ; or, je pense qu'il ne faut refacto des choses que s'il y a un réel besoin/consensus, sinon le gain est peu évident, avec les désavantages cités par @Crash-- .
Dans tous les cas, cet avis subjectif n'est pas bloquant de mon côté ;)

Reply:
trop tard, j'ai tout viré

On peut valider ?